build/variables.yml

```diff
@@ -69,9 +72,17 @@
   # Flutter version.
   FlutterVersion: '3.22.1'
 
+  # Docker container to run MobSF.
+  DockerVersion: '25.0.5'
+
+  # MobSF Auth key.
+  # This key is used to authenticate the MobSF API. since our instance is not exposed, we can use any random key even a Guid.
```

Comment:
What is _our_ instance? This code is a template to generate new apps and can be used by anybody in the world. People need to know how to access `an` instance. They're not necessarily going to have `our` instance.

If extra setup steps are required for the security scan to work, it should be indicated as a TODO comment (so that when they generate the app, they already have the todo for it).

Reply:
And that's precisely what this check is about in the PR checklist.
![image](https://github.com/user-attachments/assets/a6b67739-f7f6-48e3-b12d-040a4851e133)

Reply:
there's no extra step to do in order to get the MobSF working... i've updated the comment with a more general info

Reply:
@jeanplevesque  @Arieldelossantos I have added a bit more detail in the comment for the API Key parameter, hopefully this will make its purpose / usage a bit clearer.